lib/Core/SQLiteBuildDB.cpp

```diff
@@ -86,7 +90,11 @@ class SQLiteBuildDB : public BuildDB {
       sqlite3_close(db);
 
       // Always recreate the database from scratch when the schema changes.
+#if defined(_WIN32)
+      result = _unlink(path.str().c_str());
+#else
       result = unlink(path.str().c_str());
+#endif
```

Comment:
Same request as with `chdir`, let's define a platform independent utility function for this.